packages/kit/src/utils/routing.js

```diff
@@ -0,0 +1,65 @@
+/** @param {string} key */
+export function parse_route_key(key) {
```

Comment:
do we want to call this `parse_route_id` to align with https://github.com/sveltejs/kit/pull/4345?

Reply:
we're going to merge #4345 into this PR, so no